### PR TITLE
adding language dropdown to navbar

### PR DESCRIPTION
**Pull request by MiraAr**
PTAL

### CODE REVIEW

**Comment by emrerdem1** [comment]
LGTM! 💯

**Review by emrerdem1 — APPROVED** [review]

File: src/components/layout/nav/Navbar.jsx [hunk 1]
```diff
@@ -101,6 +116,13 @@ const Navbar = () => {
             <i className="fas fa-shopping-cart"></i>
           </NavLink>
         </div>
+        <div className="iconWrapper">
+          <i
+            class="fas fa-globe"
+            onClick={() => handleStatusOfIcons("IS_LANGUAGE_DROPDOWN_OPENED")}
```
Comment (emrerdem1) [inline src/components/layout/nav/Navbar.jsx]: Only this need to be changed into `className`.

File: src/components/header/Header.jsx [hunk 2]
```diff
@@ -15,7 +18,7 @@ export default function Header() {
           <p className="whiteLine mt-4"></p>
         </div>
         <div className="headerTextContent">
-          <h1 className="headerTitle">We Care About Your Health</h1>
+          <h1 className="headerTitle">{t("We Care About Your Health")}</h1>
```
Comment (louisrli) [inline src/components/header/Header.jsx]: this doesnt seem right, the translations are usually short keys, not full sentences?

File: src/components/locales/ar/translation.json [hunk 3]
```diff
@@ -1,3 +1,3 @@
 {
-  "omar": "عمر"
-}
+  "We Care About Your Health": "صحتكم تهمنا"
```
Comment (louisrli) [inline src/components/locales/ar/translation.json]: the key should be a short name like "title" or something, not the english version of the sentence

**Review by louisrli — CHANGES_REQUESTED** [review]

**Comment by MiraAr** [comment]
PTAL

**Review by louisrli — APPROVED** [review]

**Pull request merged**